app/models/user.rb

```diff
@@ -6,9 +6,6 @@ class User < OrganizationalUnit
 
```

Comment:
Could you add a method to get all repositories a user has access to via all his organizations.